app/experimenter/experiments/filtersets.py

```diff
@@ -265,7 +265,9 @@ def is_paused_filter(self, queryset, name, value):
     def completed_results_filter(self, queryset, name, value):
         if value:
             return queryset.exclude(
-                results_url=None, results_initial=None, results_lessons_learned=None
+                (Q(results_url="") | Q(results_url=None)),
+                (Q(results_initial="") | Q(results_initial=None)),
+                (Q(results_lessons_learned="") | Q(results_lessons_learned=None)),
```

Comment:
Minor nit: You don't need the parens around this, since they are separate parameters.